packages/@o3r/schematics/schematics/ng-add/index.ts

```diff
@@ -1,33 +1,9 @@
 import type { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
-import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
 import * as fs from 'node:fs';
 import * as path from 'node:path';
 import { lastValueFrom } from 'rxjs';
 import type { PackageJson } from 'type-fest';
-import { NodePackageName } from '@angular-devkit/schematics/tasks/package-manager/options';
-
-/**
- * Install dev dependency on your application
- *
- * Note: it should not be moved to other packages as it should run before the installation
- * of peer dependencies
- */
-class DevInstall extends NodePackageInstallTask {
-  /** @inheritDoc */
-  public toConfiguration() {
-    const installOptions = process.env && process.env.npm_execpath && process.env.npm_execpath.indexOf('yarn') === -1 ? 'npm' : 'yarn';
-    return {
-      name: NodePackageName,
-      options: {
-        command: 'install',
-        quiet: this.quiet,
-        workingDirectory: this.workingDirectory,
-        packageName: `${this.packageName!} ${installOptions === 'yarn' ? '--prefer-dev' : '-D'}`,
-        packageManager: installOptions
-      }
-    };
-  }
-}
+import { AddDevInstall } from '@o3r/schematics';
```

Comment:
`@o3r/schematics` can actually import from itself ? 
I assume the dependency is already installed when the nd-add script is being executed

Reply:
Yes, it is working only with this package because schematics support only commonjs package require